stash/src/service/SyncTransactionsService.ts

```diff
@@ -0,0 +1,37 @@
+import MangataClient from '../connector/MangataNode.js'
+import {
+  watchDepositAcceptedIntoQueue,
+  processRequests,
+} from '../scraper/L1LogScraper.js'
+import logger from '../util/Logger.js'
+import { holesky, arbitrumSepolia } from 'viem/chains'
+const ETH_CHAIN = 'Ethereum'
```

Comment:
Official names for chains are Arbitrum Sepolia and Holesky. In our lastProcessedRequestOnL2 are Arbitrum and Ethereum. That is the reason for introducing the separation in stash code as well.